sarracenia/flowcb/log.py

```diff
@@ -81,7 +138,7 @@ def after_accept(self, worklist):
                         "%s rejected: %d %s " %
                         (msg['relPath'], msg['report']['code'], msg['report']['message']))
                 else:
-                    logger.info("rejected: %s " % self._messageStr(msg))
```

Comment:
can we keep the original full message output as a debug-level log?

Reply:
already have logMessageDump  to dump the entire message instead of the default subset selection.